test/core_fusion.cpp

```diff
@@ -830,6 +830,12 @@ TEST(core_fusion, softmax_crossentropy_bprop_with_ignore_mask)
 }
 #endif
 
+// TODO(pthoreho): MLIR currently doesnot support all the op's needed for CrossEntropy+Softmax
+// this results in multiple CompiledKernels and we cannot able to safely check for certain op's
+// from the function created by user.
+// Note: remove this gaurds once we have full support for CE and Softmax through MLIR
+
+#ifndef NGRAPH_MLIR_ENABLE
```

Comment:
Instead, you can add MLIR_DISABLE_TEST to the test using this function. For example:

```
TEST(core_fusion, crossentropy)
```
-->
```
TEST(core_fusion, MLIR_DISABLE_TEST(crossentropy))
```